application/src/main/java/com/github/pieter_groenendijk/model/LoanHistory.java

```diff
@@ -0,0 +1,49 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Table(name = "loanHistory")
+public class LoanHistory {
```

Comment:
Is het niet redundant om de `LoanHistory` apart te gaan opslaan in de db? Al deze informatie in deze entiteit lijkt mij al herleidbaar uit `Loan`.

Of gaat het hier om een db view of iets dergelijks?